lib/web_ui/dev/utils.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io' as io;
+
+import 'package:path/path.dart' as path;
+
+import 'environment.dart';
+
+class FilePath {
+  FilePath.fromCwd(String filePath) : _path = path.absolute(filePath);
```

Comment:
In case you haven't noticed, there are getters called `relativeToCwd` and `relativeToWebUi`. If I rename the constructors, what should I call the getters, any idea?

Reply:
Ah, nvm then. "From" seems to convey the meaning just fine.